packages/client/src/styles/variables/colors-theme-light.ts

```diff
@@ -1,9 +1,12 @@
 export const COLOR_PREVIEW_PRIMARY = '#d6a13b';
-export const COLOR_ELEMENT_PRIMARY = '#23B7CB';
-export const COLOR_TEXT_PRIMARY = '#FFF';
+export const COLOR_ELEMENT_PRIMARY = '#23b7cb';
+export const COLOR_TEXT_PRIMARY = '#ffffff';
 export const COLOR_ELEMENT_SECONDARY = '#495057';
-export const COLOR_TEXT_SECONDARY = '#803737B6';
+export const COLOR_TEXT_SECONDARY = '#803737b6';
 export const COLOR_ELEMENT_WARN = '#dc2f55';
-export const COLOR_BACKGROUND_SECONDARY = '#D9D9D9';
+export const COLOR_BACKGROUND_SECONDARY = '#d9d9d9';
+export const COLOR_TABLE_BACKGROUND_PRIMARY = '#cadec6';
+export const COLOR_TABLE_BACKGROUND_SECONDARY = '#afcdaa';
```

Comment:
```suggestion
export const BACKGROUND_COLOR_TABLE_PRIMARY = '#cadec6';
export const BACKGROUND_COLOR_TABLE_SECONDARY = '#afcdaa';
```
https://github.com/FancyCodersYAP/UNO-fancy/pull/11/files#r1133306388

Reply:
Поправила